src/server/scripts/Outland/HellfireCitadel/HellfireRamparts/boss_vazruden_the_herald.cpp

```diff
@@ -97,7 +97,7 @@ class boss_nazan : public CreatureScript
             void IsSummonedBy(Unit* summoner) override
             {
                 if (summoner->GetEntry() == NPC_VAZRUDEN_HERALD)
-                    VazrudenGUID = summoner->GetGUID();
+                    VazrudenGUID = summoner->FindNearestCreature(NPC_VAZRUDEN, 5000)->GetGUID();//Is it possible to get the GUID directly?
```

Comment:
Its instance, you can do that via instance script

Reply:
tks, I'll do it later.

Reply:
Also for future reference, this code will cause application crash if FindNearestCreature returns null. And findNearestcreature is expensive grid call and try avoiding any grid calls in instances :P